src/Main.cpp

```diff
@@ -29,23 +32,123 @@
 #  include <direct.h>
 #endif
 
+static void openglCallbackFunction(
+        GLenum source,
+        GLenum type,
+        GLuint id,
+        GLenum severity,
+        GLsizei length,
+        const GLchar *message,
+        const void *userParam
+) {
+    // ignore non-significant error/warning codes
+    if (id == 131169 || id == 131185 || id == 131218 || id == 131204) return;
```

Comment:
Sytle nit:  Newline after if statement.

Also, do these have #defines or enum constants?  The opaque IDs aren't meaningful by themselves.

Reply:
A bit a confused about your style nit, there is a newline after the if statement, do you want it to be removed?

Reply:
So I have been looking for the meaning of these ids because honestly I just copied this code from somewhere and rather surprisingly I could not find any docs about them. So for now I have removed the if, simultaneously solving the newline issue. There should probably be some "only during debug" flag for this code anyway. If we feel this leads to too much output it's probably better to filter on severity.